docs/ldml/tr35-dates.md

```diff
@@ -2365,6 +2365,314 @@ The meaning of symbol fields should be easy to determine; the problem is determi
+
+#### Mapping to Standard Skeletons
+
+The selected fields in the field set should be converted to standard skeleton symbols according to the following table. "Standalone" should be used when the field set contains only one field. Hour is broken down by [hour cycle](#Semantic_Skeleton_Hour_Cycle), and Zone is broken down by [time zone style](#Semantic_Skeleton_Time_Zone_Style).
+
+| Semantic Field               | Long   | Medium | Short  |
+|------------------------------|--------|--------|--------|
+| Year                         | \*     | \*     | \*     |
+| Month                        | \*     | \*     | \*     |
+| Month (Standalone)           | LLLL   | LLL    | L      |
+| Day                          | \*     | \*     | \*     |
+| Weekday                      | EEEE   | EEE    | EEE    |
+| Weekday (Standalone)         | EEEE   | EEE    | EEEEE  |
+| Hour - Auto                  | C      | C      | C      |
+| Hour - H11, H12              | h      | h      | h      |
+| Hour - H23, H24              | H      | H      | H      |
+| Minute                       | m      | m      | m      |
+| Second                       | s      | s      | s      |
+| Zone - Generic               | v      | v      | v      |
+| Zone - Generic (Standalone)  | vvvv   | vvvv   | v      |
+| Zone - Specific              | z      | z      | z      |
+| Zone - Specific (Standalone) | zzzz   | zzzz   | z      |
+| Zone - Location              | VVVV   | VVVV   | VVVV   |
+| Zone - Offset                | O      | O      | O      |
+
+\* Lengths for Year, Month, and Day are taken from the [datetimeSkeleton](#dateFormats) in the Long, Medium, and Short variants. The era field, if present, should be included with the Year. For example, in en-US, CLDR 46, the datetimeSkeletons are:
+
+| Length | Calendar  | datetimeSkeleton |
+|--------|-----------|------------------|
+| Long   | Gregorian | yMMMMd           |
+| Medium | Gregorian | yMMMd            |
+| Short  | Gregorian | yyMd             |
+| Long   | Japanese  | GyMMMMd          |
+| Medium | Japanese  | GyMMMd           |
+| Short  | Japanese  | GGGGGyMd         |
+
+This means that the Year, Month, and Day semantic field mapping in en-US should be:
+
+| Semantic Field | Calendar  | Long   | Medium | Short  |
+|----------------|-----------|--------|--------|--------|
+| Year           | Gregorian | y      | y      | yy     |
+| Month          | Gregorian | MMMM   | MMM    | M      |
+| Day            | Gregorian | d      | d      | d      |
+| Year           | Japanese  | Gy     | Gy     | GGGGGy |
+| Month          | Japanese  | MMMM   | MMM    | M      |
+| Day            | Japanese  | d      | d      | d      |
+
+##### <a name="Semantic_Year_Style_Skeleton_Variations" href="#Semantic_Year_Style_Skeleton_Variations">Year Style Skeleton Variations</a>
+
+The [year style](#Semantic_Skeleton_Year_Style) should change the skeleton for all lengths as follows:
+
+- Auto: No change from datetimeSkeleton
```

Comment:
Add a note: the datetimeSkeleton may already be y; and may include G already, subject to locale preferences.

Reply:
Similar notes elsewhere, just for clarity.